README.md

```diff
@@ -53,7 +53,9 @@ your component.
 - `group`: component group
 - `type`: component folder type - content, form, structure
 - `options.js`: whether to create an empty JS lib for the component (shared with CSS lib)
+- `options.jstext`: whether to create the js text file within the clientlib. Set to `false` when this file is not needed within your clientlib.
```

Comment:
honestly I never think of this as a "text" file - its really a mapping file.  I really think we should name this `jstxt` not `jstext` and should describe it as what developers would know it as - "js.txt mapping file" or even just "js.txt file" rather than "js text file".

Same as with the CSS file.